x-pack/plugins/security_solution/public/cases/attachments/external_reference_event.test.tsx

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+
+import AttachmentContentEvent from './external_reference_event';
+import { useNavigation } from '@kbn/security-solution-navigation/src/navigation';
+
+jest.mock('@kbn/security-solution-navigation/src/navigation', () => {
+  return {
+    useNavigation: jest.fn(),
+  };
+});
+
+describe('AttachmentContentEvent', () => {
+  const mockNavigateTo = jest.fn();
+
+  const mockUseNavigation = useNavigation as jest.Mocked<typeof useNavigation>;
+  (mockUseNavigation as jest.Mock).mockReturnValue({
+    getAppUrl: jest.fn(),
+    navigateTo: mockNavigateTo,
+  });
+
+  const defaultProps = {
+    externalReferenceMetadata: {
+      command: 'isolate',
```

Comment:
If this is to be `IExternalReferenceMetaDataProps` isn't `comment` missing here?

Reply:
Hey, good catch :) This is the reason why we had the discussion above, currently I do not expect all the metadata here. But... I am going to change the expected props type, so it requires whole IExternalReferenceMetaDataProps and will adjust that in tests :)